mava/systems/ff_ippo.py

```diff
@@ -450,12 +454,15 @@ def run_experiment(_config: DictConfig) -> None:
     """Runs experiment."""
     config = copy.deepcopy(_config)
 
+    # Calculate total timesteps.
```

Comment:
the comment is wrong here this is getting the number of devices and this comment should be before line 477 (same for other files)